build.gradle

```diff
@@ -1173,6 +1174,20 @@ project(':forge') {
         }
     }
 
+    tasks.register('checkAll') {
+        dependsOn 'checkLicenses'
+        group = 'checks'
+    }
+    tasks.register('checkAllAndFix') {
+        dependsOn 'findFieldInstanceChecks', 'checkLicenses'
+        group = 'checks'
+    }
+    afterEvaluate {
+        tasks.withType(CheckTask).all {
+            tasks.getByName(it.mode.get() === CheckMode.FIX ? 'checkAllAndFix' : 'checkAll').dependsOn(it)
+        }
+    }
```

Comment:
Why does this use `afterEvaluate` and the eager methods (`all`, `getByName`) instead of the equivalent lazy task configuration API methods (`configureEach`, `named(...).configure`)?